project.clj

```diff
@@ -4,4 +4,7 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.8.0"]
-                 [honeysql "0.6.3"]])
+                 [honeysql "0.9.2"]]
```

Comment:
The latest available version of honeysql is 0.9.1, please fix this.

Reply:
Yes I can fix it, however I think that this patch does not make sense without the other in `honeysql`, so we might need to wait for that to be released (has been merged)

Reply:
Okay. Will consider this once that 0.9.2 has been released. 

Also, thanks for raising the PR.

Reply:
@krish8664 honeysql got released :tada: :smile:

Reply:
@arichiardi Finally :), I'll merge this by Monday.

Reply:
@krish8664 Great! I also created an `npm` org where we can deploy this package as well. If you have a login/username I can add you or I can publish myself, up to you :smile: 

https://www.npmjs.com/settings/honeysql/packages

Reply:
ping :+1: :smile: :100:

Reply:
ah. sorry. merging this. 

You can publish the npm package 👍